index.html

```diff
@@ -15,15 +15,107 @@
             <img src="sprites/Union.png" alt="">
         </nav>
         <header>
-            <h1 class="tittle">Wander Gonzalez</h1>
-            <h2 class="tittle-2">Software Developer</h2>
+            <h1 class="title">Wander Gonzalez</h1>
+            <h2 class="title-2">Software Developer</h2>
         </header>
         <div>
             <p class="text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
         </div>
-        <img class="vector-img" src="sprites/Vector4.png" alt="End of section 1">
+        
     </section>
-
-    </div>
+    <img class="vector-img" src="sprites/Vector4.png" alt="End of section 1">
+    <section id="section-2">
+        <ul>
+            <li class="list-title" id="language-t">Language</li>
+            <li class="list-content`" id="language-l"><ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul></li>
+            <li class="list-title" id="frameworks-t">Frameworks</li>
+            <li class="list-content" id="frameworks-l"><ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+            </ul></li>
+            <li class="list-title" id="skills-t">Skills</li>
+            <li class="list-content" id="skills-l"><ul>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul></li>
+        </ul>
+    </section>
+    <img class="vector-img" src="sprites/Vector4.png" alt="End of section 2">
+    <section id="section-3">
+        <nav id="social-media-icons">
+            <a href="#"><img src="sprites/Disabled.png" alt="Github icon"></a>
+            <a href="#"><img src="sprites/Disabled-1.png" alt="Linkdin icon"></a>
+            <a href="#"><img src="sprites/Enabled.png" alt="Angelist icon"></a>  
+            <a href="#"><img src="sprites/Disabled-2.png" alt="Twitter icon"></a>
+            <a href="#"><img src="sprites/Disabled-3.png" alt="Medium icon"></a>
+        </nav>
+        <div>
+            <img src="sprites/Group55.png" alt="Down Arrow"><h2>Work</h2>
+        </div>
+        <h2 class="title">MY RECENT WORK</h2>
```

Comment:
- [ ] Please use this image below compared to the Figma and make necessary changes 
- Please make some changes to the font size of these in the image below to match the design in the Figma

<img width="309" alt="toitoi" src="https://user-images.githubusercontent.com/79058364/159075624-b33e94e7-4807-4914-a0a8-3c0b2966a7b8.png">

#### Figma design

<img width="226" alt="figtoi" src="https://user-images.githubusercontent.com/79058364/159076142-42dfa8ce-4f9a-48ae-9d20-5cb4f6113530.png">